packages/ui/client/components/explorer/ExplorerItem.vue

```diff
@@ -146,29 +176,46 @@ const highlighted = computed(() => {
         {{ duration > 0 ? duration : '< 1' }}ms
       </span>
     </div>
-    <div v-if="type === 'file'" gap-1 justify-end flex-grow-1 pl-1 class="test-actions">
+    <div gap-1 justify-end flex-grow-1 pl-1 class="test-actions">
       <IconAction
         v-if="!isReport && failedSnapshot"
         v-tooltip.bottom="'Fix failed snapshot(s)'"
         data-testid="btn-fix-snapshot"
         title="Fix failed snapshot(s)"
-        icon="i-carbon-result-old"
+        icon="i-carbon:result-old"
         @click.prevent.stop="updateSnapshot(task)"
       />
-      <IconAction
-        v-tooltip.bottom="'Open test details'"
-        data-testid="btn-open-details"
-        title="Open test details"
-        icon="i-carbon-intrusion-prevention"
-        @click.prevent.stop="onItemClick?.(task)"
-      />
-      <IconAction
+      <VueTooltip
+        placement="bottom"
+        class="w-1.4em h-1.4em op100 rounded flex"
+        :class="showDetailsClasses"
+      >
+        <IconButton
+          data-testid="btn-open-details"
+          icon="i-carbon:intrusion-prevention"
+          @click.prevent.stop="showDetails"
+        />
+        <template #popper>
+          <div v-if="disableShowDetails" class="op100 gap-1 p-y-1" grid="~ items-center cols-[1.5em_1fr]">
+            <div class="i-carbon:information-square w-1.5em h-1.5em" />
+            <div>{{ showDetailsTooltip }}: this feature is not available, you have disabled <span class="text-[#add467]">includeTaskLocation</span> in your configuration file.</div>
+            <div style="grid-column: 2">
+              Clicking this button the code tab will position the cursor at first line in the source code since the UI doesn't have the information available.
+            </div>
+          </div>
+          <div v-else>
+            {{ showDetailsTooltip }}
+          </div>
+        </template>
+      </VueTooltip>
+      <IconButton
         v-if="!isReport"
         v-tooltip.bottom="'Run current test'"
         data-testid="btn-run-test"
         title="Run current test"
         icon="i-carbon:play-filled-alt"
         text-green5
+        :disabled="type !== 'file'"
```

Comment:
> added disabled run test/suite for future usage: we'll need to change the logic to allow run individual tests/suites.

Sorry if I am late, but could you please clarify why the ability to run individual test was disabled?

Is is something that is coming in the future? Is there a feature request for that or should I open one?

Thanks!

Reply:
This was never supported. It is possible to support - feel free to create an issue or a PR